sql/setup.sql

```diff
@@ -1,2 +1,20 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+DROP table if exists cats;
+create table cats (
+    id BIGINT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR NOT NULL,
+    type VARCHAR NOT NULL
+    -- URL VARCHAR NOT NULL,
+    -- year INT NOT NULL,
+    -- lives INT NOT NULL
+);
```

Comment:
We will also want to add in url, year, lives, and isSidekick into our sql database.